adapters/appnexus/appnexus.go

```diff
@@ -403,11 +404,11 @@ func (a *adapter) findIabCategoryForBid(bid *appnexusBidExt) (string, bool) {
 }
 
 func appendMemberId(uri string, memberId string) string {
-	if strings.Contains(uri, "?") {
-		return uri + "&member_id=" + memberId
-	}
-
-	return uri + "?member_id=" + memberId
+	u, _ := url.Parse(uri)
```

Comment:
Please do not ignore errors. I recommend moving `url.Parse` into the `Builder` method. You can safely make a shallow clone of the `URL` object here instead and proceed with the query string manipulation.

Reply:
Thank you for the suggestion, Scott! I have made the changes in [61090f7](https://github.com/prebid/prebid-server/pull/2747/commits/61090f76f85cced2fff19791b0e43c06b45e2fba)